lib/osf-components/addon/components/file-browser/template.hbs

```diff
@@ -1,6 +1,10 @@
 <FileBrowser::AddNew::Upload
     @manager={{@manager}}
     @enable={{and @manager.currentFolder.userCanUploadToHere @enableUpload}}
+    @dragEnter={{fn (mut this.dragging) true}}
+    @dragOver={{fn (mut this.dragging) true}}
+    @dragLeave={{fn (mut this.dragging) false}}
+    local-class='{{if this.dragging 'DraggingHighlight'}}'
```

Comment:
It looks like this `.DraggingHighlight` class sticks around even after a successful drag+drop file add. Is it possible to toggle `this.dragging` after the files get added?